src/app/(post-login)/submission/program/[shortName]/clinical-submission/components/ProgressBar.tsx

```diff
@@ -98,11 +101,22 @@ const ProgressBar: FC<ProgressBarProps> = ({
       ? "success"
       : "disabled",
   };
+  const pendingApprovalWidth = `${approvalBarWidth || 100}px`;
```

Comment:
Is this something that needs to be a prop to `ProgressBar` comp? 
seems like it may be a case of "pending approval" progress item is always larger

Reply:
This is a carryover from platform-ui
It is used here: https://github.com/icgc-argo/platform-ui/blob/develop/components/pages/submission-system/program-dashboard/ProgramWorkspaceStatus/index.tsx#L142